files/image_config/platform/rc.local

```diff
@@ -207,6 +207,17 @@ program_console_speed()
     systemctl daemon-reload
 }
 
+request_dpu_midplane_ip()
+{
+    local INTERFACE="eth0-midplane"
+    if ip link show "$INTERFACE" 2> /dev/null; then
+        echo "dhclient -r $INTERFACE"
+        /usr/sbin/dhclient -r "$INTERFACE"
```

Comment:
@vivekrnv what if the server is not ready or dhclient reports any error? Could you capture that?